### PR TITLE
Add PEP8 CI & fix all repo with PEP8

### PR DESCRIPTION
**Pull request by OnlyDeniko**

### CODE REVIEW

File: daal4py/sklearn/neighbors/tests/test_kneighbors.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#===============================================================================
+# ===============================================================================
```
Comment (PetrovKP) [inline daal4py/sklearn/neighbors/tests/test_kneighbors.py:1]: copyrights shouldn't change!

**Review by PetrovKP — COMMENTED** [review]
Great work! Thanks you

**Review by PetrovKP — APPROVED** [review]

**Comment by agorshk** [comment]
I suggest next time also provide information how to fix new formatting (with scripts/tools)

**Comment by PetrovKP** [comment]
@agorshk 

All errors why failed, in which file, in which line are described in the log when the job falls. What do you suggest adding?
locally you can also reproduce with the following commands:

```bash
pip install flake8
flake8 --ignore=E265,E722,E402,F401,F403 --max-line-length=90 --count
```

**Comment by agorshk** [comment]
@PetrovKP Yes, I see it from ci.yml I suppose that all issues that had been found by flake8 were fixed by other tools or scripts. It'll be convenient to add information about it in description or somewhere, the idea is not to fix errors manually.

**Comment by OnlyDeniko** [comment]
@agorshk, a lot of problems were fixed by autopep8 (https://pypi.org/project/autopep8/) and pep8 (https://pypi.org/project/pep8/). But this tools is not guarantee that your code will be clean up. Some troubles you have to fix manually.

**Pull request merged**